.gitignore

```diff
@@ -1,5 +1,11 @@
+# Backend stuff
+.idea/
+.vagga/
+runtime/data/*
 node_modules/
-log.txt
 config/config.js
+data.xml
+log.txt
+# Frontend stuff
 static/js/app.min.js
-static/css/style.min.css
+static/css/style.min.css
```

Comment:
Thank you for the pull request! Nice to see, that project is still in demand. I have a few remarks, and it would be cool if you fix that.

It is a common practice to put new line character at the end of file. You can read more [here](https://help.github.com/articles/dealing-with-line-endings/). 

If you don't put new line at the end of file, you see cicrle with arrow near last line at github.